### PR TITLE
#74 Notification badge

### LINKED ISSUE #74: Add notification badge when new info about requests is available
As a user I want to get a notification if there is new information about my received and sent requests

acceptance criteria: 

notification should appear when:
- [ ] current user receives a new request
- [ ] current users sent request gets answered

### PR DESCRIPTION
**Pull request by Erikisak**
Notification badge added 
Badge will show in header when request i recieved
Closes #74

### CODE REVIEW

**Review by mirrenil — COMMENTED** [review]
Nice work! Can I assume that the badge disappears when a request has been answered?

**Comment by Erikisak** [comment]
> Nice work! Can I assume that the badge disappears when a request has been answered?

Worked in Linneas branch so I also assumed that!

**Review by mirrenil — DISMISSED** [review]

**Review by liinneea-a — CHANGES_REQUESTED** [review]
Nice!! Just add a key prop :)

File: src/Components/Header.tsx [hunk 1]
```diff
@@ -19,10 +19,12 @@ import { useAuth } from "../Contexts/AuthContext";
 import MobileHeader from "../Assets/Images/mobileHeader.svg";
 import { ToastContainer } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
+import { useUser } from "../Contexts/UserContext";
 
 const Header = () => {
   const [menuOpen, setMenuOpen] = useState(false);
   const { currentUser, logout } = useAuth();
+  const { myReceivedRequests, mySentRequests } = useUser();
```
Comment (mirrenil) [inline src/Components/Header.tsx:27]: is "mySentRequests" used somewhere?

**Comment by liinneea-a** [comment]
A badge is now rendered as many times as user has requests. There should only be one badge (if user has requests)

**Review by AdaHep — APPROVED** [review]
Looks good :) mySentRequests on line  27 is still not used and can be removed 👍

**Review by hazem-89 — APPROVED** [review]

**Review by mirrenil — APPROVED** [review]
😮‍💨

**Review by liinneea-a — APPROVED** [review]

**Pull request merged**